instaclone/app/post/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import BigInteger, Integer
+from sqlalchemy.orm import Mapped, mapped_column
+from instaclone.database.common import Base
+
+
+class Post(Base):
+    __tablename__ = "posts"
+
+    # user_id
+    user_id: Mapped[int] = mapped_column(BigInteger)
+    # story_id
+    story_id: Mapped[int] = mapped_column(BigInteger)
```

Comment:
```suggestion
    post_id: Mapped[int] = mapped_column(BigInteger)
```
혹시 의도적으로 story_id로 설정하신걸까요? Post attribute가 아직 반영 안된 것 같네요.

Reply:
앗..! post 작성하다가 자리를 옮겨서,, comment 구성까지 누락되었네요..! 수정해서 올리겠습니다!